opds2_import.py

```diff
@@ -686,13 +731,16 @@ def _get_publications(feed):
         :return: An iterable list of publications containing in the feed
         :rtype: Iterable[opds2_ast.OPDS2Publication]
         """
-        for publication in feed.publications:
-            yield publication
-
-        for group in feed.groups:
-            for publication in group.publications:
+        if feed.publications:
```

Comment:
This is also a very important part since `webpub-manifest-parser` for some reason don't set empty `publications` and `groups` properties to empty lists for some reason. I'll look into this later